rubix-bookkeeper/src/main/java/com/qubole/rubix/bookkeeper/BookKeeper.java

```diff
@@ -412,7 +412,7 @@ public FileInfo getFileInfo(String remotePath) throws TException
         return info;
       }
       catch (Exception e) {
-        log.error(String.format("Could not fetch FileStatus from remote file system for %s : %s", remotePath), e);
+        log.error(String.format("Could not fetch FileStatus from remote file system for %s : %s", remotePath, e));
```

Comment:
keep `e` an argument to log.error. Change the string format instead.

Reply:
fixed.